src/assets/scss/component/_custom-form-styles.scss

```diff
@@ -58,502 +55,494 @@ $formContainerBorderColor: #ccc;
-.ss-required-asterisk {
-  color: $error
-}
-
-.required-message {
-  display: none;
-}
-
-.ss-section-title {
-  background-color: $sectionTitleBackground;
-  padding: 0.4em;
-  margin: 2em -0.4em 0;
-}
-.ss-section-description {
-  margin-top: 0.5em
-}
-.ss-page-title {
-  margin: 0;
-  padding: 0;
-}
-.ss-formwidget-container:focus {
-  outline: none
-}
-.ss-item {
-  margin: $bodyLineHeight*1em 0
-}
-.ss-image, .ss-image-container {
-  max-width: 100%
-}
-.ss-image:focus {
-  outline: none
-}
-.img-menu .jfk-button-collapse-left {
-  margin-right: 0
-}
-.ss-edit-link, .ss-edit-link:active, .ss-edit-link:visited, .ss-edit-link:hover, .ss-edit-link:link {
-  background: url(//www.gstatic.com/docs/spreadsheets/forms/edit_dark.png) 7px 8px no-repeat #1f1f1d;
-  border: 1px solid #fff;
-  color: #fff;
-  float: right;
-  font: normal 13px arial,sans-serif;
-  padding: 7px 7px 7px 27px;
-  text-decoration: none;
-}
-
-.disclaimer .ss-terms a:link {
-  color: $disclaimerLinkColor;
-  text-decoration: none;
-}
-.disclaimer .ss-terms a:hover {
-  text-decoration: underline
-}
-.disclaimer .ss-terms a:active {
-  color: $disclaimerLinkActiveColor
-}
-.disclaimer .ss-terms a:visited {
-  color: $disclaimerLinkVisitedColor
-}
-
-.disclaimer {
-  color: $disclaimerColor;
-  font: normal 11px arial,sans-serif;
-  margin-top: 11px;
-}
-.disclaimer-separator {
-  border-top: 1px solid $disclaimerSeparatorBorderColor
-}
-
-.disclaimer .ss-logo-link {
-  color: $disclaimerLogoLinkColor;
-  text-decoration: none;
-}
-.powered-by-logo {
-  display: inline-block;
-  vertical-align: bottom;
-}
-.powered-by-text {
-  line-height: 16px
-}
-.ss-terms {
-  display: inline-block;
-  line-height: 20px;
-  width: 80%;
-  text-align: center;
-}
-.ss-submit-message {
-  font: normal 13px arial,sans-serif;
-  margin-bottom: 20px;
-}
-
-.ss-form-container {
-  background-color: $formContainerColor;
-  border: 1px solid $formContainerBorderColor;
-  margin: 0.5em auto 0;
-  padding: 1em;
-  width: 45em;
-}
-
-.ss-editing-response {
-  border: 1px solid $editingResponseBorderColor;
-  margin: 1em auto;
-  padding: 0 1em;
-  width: 43em;
-}
-
-.ss-confirmation-banner, .ss-editing-response {
-  background-color: $confirmationBannerBackground
-}
-.errorbox-bad {
-  border: 2px solid $error;
-  background-color: $errorBoxBadBorderColor;
-  padding: 2px;
-}
-.error-message {
-  display: none;
-}
-
-.errorheader {
-  color: $error
-}
-input#emailReceipt {
-  margin-left: 0;
-  margin-top: 0;
-  vertical-align: middle;
-}
-.no-chrome .powered-by-logo, .no-chrome .ss-legal, .no-chrome .ss-edit-link {
-  display: none
-}
-.ss-prepopulate-input, .ss-submit-url-input {
-  width: 100%
+.google-form {
+  html .goog-inline-block,
+  *:first-child+html .goog-inline-block {
+    display: inline;
+  }
+  body {
+    color: $baseColor;
+    font-family: $bodyFontFamily;
+    font-weight: 400;
+    margin: 0;
+    line-height: $bodyLineHeight;
+    padding: 0 1em;
+    background-color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.